SetupDataPkg/Include/Library/ConfigKnobShimLib.h

```diff
@@ -0,0 +1,46 @@
+/** @file ConfigKnobShimLibCommon.h
+  Common library interface for the autogen XML config header to call into to fetch a config value.

```

Comment:
Comment is out of date, should be: `Library interface for an OEM config policy creator to call into to fetch overrides for config values.`

Reply:
Updated.